### PR TITLE
Move anago subcommands to own directory

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
We should stick to a naming convention which makes it easy to
distinguish "throw away" anago subcommands vs first-class krel
ones.
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by justaugustus — CHANGES_REQUESTED** [review]
Thanks for following up on this!
Can we go a step further and drop all of the anago commands into `cmd/krel/cmd/anago/`?

File: cmd/krel/cmd/root.go [hunk 1]
```diff
@@ -68,6 +69,8 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&rootOpts.cleanup, "cleanup", false, "cleanup flag")
 	rootCmd.PersistentFlags().StringVar(&rootOpts.repoPath, "repo", filepath.Join(os.TempDir(), "k8s"), "the local path to the repository to be used")
 	rootCmd.PersistentFlags().StringVar(&rootOpts.logLevel, "log-level", "info", "the logging verbosity, either 'panic', 'fatal', 'error', 'warn', 'warning', 'info', 'debug' or 'trace'")
+
+	rootCmd.AddCommand(anago.AnagoCmd)
```
Comment (saschagrunert) [inline cmd/krel/cmd/root.go:73]: Init functions are triggered "the other way around" which means that we now have to explicitly add the subcommand there.

**Comment by saschagrunert** [comment]
> Thanks for following up on this!
> Can we go a step further and drop all of the anago commands into `cmd/krel/cmd/anago/`?

Done :heavy_check_mark:

**Review by hasheddan — APPROVED** [review]
/lgtm

**Pull request merged**